backend/infrahub/graphql/manager.py

```diff
@@ -578,7 +579,7 @@ def generate_graphql_object(self, schema: MainSchemaTypes, populate_cache: bool
         for attr in schema.local_attributes:
             attr_kind = get_attr_kind(schema, attr)
             attr_type = self.get_type(name=get_attribute_type(kind=attr_kind).get_graphql_type_name())
-            main_attrs[attr.name] = graphene.Field(attr_type, required=not attr.optional, description=attr.description)
+            main_attrs[attr.name] = graphene.Field(attr_type, description=attr.description)
```

Comment:
I wonder if we should have something in the description to indicate that the field "can be required".

Something like:

```python
req = "" if attr.optional else " (required)"
main_attrs[attr.name] = graphene.Field(attr_type, description=f"{attr.description}{req}")
```
I'm not certain that it would be helpful?